### ISSUE TITLE
uses lucky eggs

### ISSUE THREAD

**Issue by wicker558** [Ar1i/PokemonGo-Bot#2609]
Hello I have problem with this bot that it automatically used my eggs, I had both options that use the eggs unchecked, I think this is a major bug and should be fixed.

**Comment by kazzaluis** [comment]
can you put a screen shot of the bot using the lucky egg and uncheck the option and screenshot of the game while the egg is active ?

**Comment by tylerkc1130** [comment]
i think the config have bugs. Automatically change to use egg and dont hatch the eggs

**Comment by tylerkc1130** [comment]
change to lucky eggs*

**Comment by kazzaluis** [comment]
@Ar1i  eggs bug

**Comment by Sobro111** [comment]
ar1i what about "cant find any pokestop in a 5000 range! 

what should we do to fix that

**Comment by Kaizhoul** [comment]
Yes i have a problem "cant find anyblablabla" it's error of my connection "i use proxy" or it's error of bot ?

**Comment by Shak4zoo** [comment]
![bg](https://cloud.githubusercontent.com/assets/20952327/17562235/39147a9e-5f2a-11e6-99e4-bb2b906fe604.png)
and how this problem svp !!

**Comment by mehfoozzz** [comment]
yes bro bot is not hatching app... and please add pokemon ssniping option.. its an humble request...

**Comment by xRealMaks** [comment]
@mehfoozzz, it does in the newest verison.

**Comment by mehfoozzz** [comment]
thanks @xRealMaks ... its working in update....

and please add pokemon sniping in bot
